package.json

```diff
@@ -69,6 +69,7 @@
     "conventional-commit-types": "^2.1.0",
     "copy-webpack-plugin": "^1.1.1",
     "cz-conventional-changelog": "^1.2.0",
+    "d3": "^3.5.17",
```

Comment:
Does cola not support d3 v4? I have been using d3 v4 for some other things I'm hoping to get in Candela.